### ISSUE TITLE
Unable to see entire history of bank transfers

### ISSUE THREAD

**Issue by CGreen92** [jmfernandes/robin_stocks#308]
Found Additional pages.
Additional pages exist but could not be loaded.

**Comment by makmin4u** [comment]
Yes, I am having the same issue. @CGreen92 were you able to resolve the issue? Anyone else able to resolve this?

**Comment by makmin4u** [comment]
https://github.com/jmfernandes/robin_stocks/issues/209#issuecomment-807902344

I resolved it using this fix. Thanks @zot24
